Integrations/integration-aria.yml

```diff
@@ -0,0 +1,2824 @@
+            return self._do_request(data)
+    
+    
+        def unblock_dest_subnet(self, rule_name: str, instance_id:str=None, label_sia_group: str=None,
+                              label_sia_name: str=None, label_sia_region: str=None) -> requests:
+            """
+            :param rule_name:
+            :param instance_id:
+            :param label_sia_group:
+            :param label_sia_name:
+            :param label_sia_region:
+            :return:
+            """
+            res = self._remove_rule(rule_name, "dst-subnet", instance_id, label_sia_group, label_sia_name, label_sia_region)
+            return res
+    
+    
+        def record_dest_subnet(self, target_ip: str, vlan_id: str, rule_name: str, sia_interface: str=None,
+                               transport_type: str=None, tti_index: str=None, aio_index: str=None, trigger_type: str=None,
+                               trigger_value: str=None, instance_id: str=None, label_sia_group: str=None,
+                               label_sia_name: str=None, label_sia_region: str=None) -> requests:
+            """
+            :param target_ip:
+            :param vlan_id:
+            :param rule_name:
+            :param sia_interface:
+            :param transport_type:
+            :param tti_index:
+            :param aio_index:
+            :param trigger_type:
+            :param trigger_value:
+            :param instance_id:
+            :param label_sia_group:
+            :param label_sia_name:
+            :param label_sia_region:
+            :return:
+            """
+    
+            if sia_interface is None or sia_interface != "B":
+                sia_interface = "A"
+            rule = "{}: REDIRECT-VLAN {} {}".format(self._process_ip_address(target_ip), sia_interface, vlan_id)
+            if transport_type is not None:
+                if tti_index is None or aio_index is None or trigger_type is None or trigger_value is None:
+                    raise Exception("Please provide tti_index, aio_index, trigger_type and trigger_value "
+                                    "to use {} to send an alert.".format(transport_type))
+                rule += ", "
+                rule += self._build_alert_instruction(transport_type, int(tti_index), int(aio_index), trigger_type, int(trigger_value))
+            rule += ", END"
+            data = self._generate_named_rule_data(rule_name, "dst-subnet", rule, "add", instance_id,
+                                                  label_sia_group, label_sia_name, label_sia_region)
+            return self._do_request(data)
+    
+    
+        def stop_recording_dest_subnet(self, rule_name: str, instance_id: str=None, label_sia_group: str=None,
+                                       label_sia_name: str=None, label_sia_region: str=None) -> requests:
+            """
+            :param rule_name:
+            :param instance_id:
+            :param label_sia_group:
+     
```

Comment:
Hi Itay, I feel like I can't use return_outputs in the test-modules functions. Meanwhile, How can I return context data when the command return error, looks like return_error() function can only display strings.

Reply:
correct, for the `test-module` you would still have to use `demisto.results('ok')`
`return_error` should be called in case an exception should be raised, and if that's the case, there shouldn't be really any outputs returned, other then an error message
why would you like to return context outputs in that case?

Reply:
That's because in multiple SIA cases. A fail is defined if there is one or more SIA fails. So return context data can help users to know which one success and which one fails. I can convert it to a string anyway. I pushed most of your fixed. I just want it to run at night, not finished for all changes so don't need to review everytime. Thanks again for your review.

Reply:
👍